app/services/scraper/prerequisites_tree.rb

```diff
@@ -0,0 +1,97 @@
+module Scraper::PrerequisitesTree
```

Comment:
nit: I think I prefer `PrerequisitesTreePage` to indicate that's a page object 🙂

Reply:
👍  changed in 6ef8f8b